projects/mystic-finance/index.js

```diff
@@ -1,11 +1,29 @@
-const {CONFIG } = require("./constants.js")
-const { aaveExports } = require("../helper/aave");
-const methodologies = require("../helper/methodologies");
+const { getMorphoVaultTvl } = require('../helper/morpoho');
```

Comment:
can we just keep the vaults adapter in this file and add myplume in a new file?

Reply:
myPlume is in the `mystic-finance-myplume/index.js` file

Reply:
I mean can we not move the vaults TVL to a new adapter file, keep the same file name, otherwise it'll cause us issues

Reply:
Do you mean having this?
- mystic-finance/index.js (vault tvl)
- mystic-finance-myPlume/index.js (lst tvl)